accounts/src/main/java/eu/kevin/accounts/bankselection/providers/DefaultCountryIsoProvider.kt

```diff
@@ -0,0 +1,22 @@
+package eu.kevin.accounts.bankselection.providers
+
+import android.content.Context
+import android.content.pm.PackageManager
+import android.telephony.TelephonyManager
+import eu.kevin.common.extensions.getCurrentLocale
+
+internal class DefaultCountryIsoProvider(
+    private val context: Context,
+    private val telephonyManager: TelephonyManager
+) {
+    fun getDefaultCountryIso(): String {
+        if (
+            context.packageManager.hasSystemFeature(PackageManager.FEATURE_TELEPHONY) &&
+            telephonyManager.phoneType != TelephonyManager.PHONE_TYPE_CDMA
+        ) {
+            return telephonyManager.networkCountryIso
```

Comment:
Should we be using `simCountryIso` here instead? Assuming I have a Polish SIM card but I'm travelling to Germany. Let's say I'm using some local German app that uses the SDK. I would still probably like to see a selection of my PL banks and not German ones to make a payment?

Or is there another reason?

Reply:
I think your proposal makes more sense. The user's bank country is more likely to be the same as SIM country. Will change it

Reply:
Looks fine but will call @aurimas-zarskis for double checking this thought process of mine : )

Reply:
Yup, I think we were talking about using sim country specifically